front/src/common/redux/forgot-password/forgot.password.api.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios';
+
+export async function forgotPassswordApi (request) {
```

Comment:
Please, don't pass actual action, pass only data. API function should not know anything about redux

Reply:
done